tests/run-make/compiler-lookup-paths/rmake.rs

```diff
@@ -0,0 +1,103 @@
+// Since #19941, rustc can accept specifications on its library search paths.
+// This test runs Rust programs with varied library dependencies, expecting them
+// to succeed or fail depending on the situation.
+// The second part of the tests also checks that libraries with an incorrect hash
+// fail to be used by the compiler.
+// See https://github.com/rust-lang/rust/pull/19941
+
+use run_make_support::fs_wrapper;
+use run_make_support::{rmake_out_path, rustc};
```

Comment:
Note: I think this needs to be updated to not use `rmake_out_path` anymore since we changed tests to have CWD be `rmake_out/` with input files copied over.

Reply:
@rustbot author